src/app/mesh/qgsmeshrendereractivedatasetwidget.cpp

```diff
@@ -185,6 +185,11 @@ QString QgsMeshRendererActiveDatasetWidget::metadata( QgsMeshDatasetIndex datase
   const auto options = gmeta.extraOptions();
   for ( auto it = options.constBegin(); it != options.constEnd(); ++it )
   {
+    if ( it.key() == QStringLiteral( "classification" ) )
+    {
+      msg += QStringLiteral( "<tr><td>Classified values</td></tr>" );
+      continue;
+    }
```

Comment:
i do not think it is necessary to treat it in special way..., i would remove this at all

Reply:
This metadata is very long and leads to a display issue on the styling widget (the width of the widget  take the width of the meta data line). Three solution :
- format the dataset to have it on several lines
- not display the classification at all
- only display that the dataset group is classified (so the user could know why the value are int and not the real value)

I have implemented the last one.

Reply:
ok